packages/shared/src/store/syncQueue.ts

```diff
@@ -12,7 +12,7 @@ interface SyncOperation {
 export class QueueClearedError extends Error {}
 
 class SyncQueue {
-  concurrency = 2;
```

Comment:
Does this actually work? I wouldn't expect it to. On a fresh install I have 131 threads, they just synchronize one-by-one.